src/test/java/cloud/eppo/EppoClientTest.java

```diff
@@ -201,26 +192,26 @@ private <T> void assertAssignment(
 
     if (assignment instanceof Boolean) {
       assertEquals(
-          failureMessage, expectedSubjectAssignment.getAssignment().booleanValue(), assignment);
+          expectedSubjectAssignment.getAssignment().booleanValue(), assignment, failureMessage);
```

Comment:
jupiter flavored asserts have the error message at the end (which I like more anyways 🤷‍♂️ )